fenzo-core/src/main/java/com/netflix/fenzo/AutoScaler.java

```diff
@@ -197,6 +199,10 @@ private boolean shouldScaleDown(long now, ScalingActivity prevScalingActivity, A
         return shouldScaleNow(false, now, prevScalingActivity, rule);
     }
 
+    private boolean shouldScaleDownInactive(long now, ScalingActivity prevScalingActivity, AutoScaleRule rule) {
+        return now > (Math.max(activeVmGroups.getLastSetAt(), prevScalingActivity.inactiveScaleDownAt) + rule.getCoolDownSecs() * 1000);
```

Comment:
I am keeping here the same checks as for the regular scale down. This is not probably relevant here, but should not be harmful as well. Removing this check may result in many more aggressive scale down operations, potentially overlapping.